drivers/overlay/ov_network.go

```diff
@@ -601,6 +618,9 @@ func (n *network) initSubnetSandbox(s *subnet, restore bool) error {
 		}
 	} else {
 		if err := n.setupSubnetSandbox(s, brName, vxlanName); err != nil {
+			// The error in setupSubnetSandbox could be a temporary glitch. reset the
+			// subnet once object to allow the setup to be retried on another endpoint join.
+			s.once = &sync.Once{}
```

Comment:
@sanimej Reviewing the restore path, I think if `restoreSubnetSandbox` fails for any reason, I think we will end up in the same issue.
And the current fix will not help, because your changes currently performs `deleteInterface` for vxlan device only if `AddInterface` fails. But if the `restoreSubnetSandbox` fails, then the failure will happen in `createVxlan` call and there is no recovery for that.

Do you think it is valid ?